### PR TITLE
Scripts, troubleshooting, Dell

### PR DESCRIPTION
**Pull request by thorsteneb**
Added to troubleshooting and compatibility; link to scripts that support hw transcode

### CODE REVIEW

**Comment by thorsteneb** [comment]
Adding what we are learning about TrueNAS Core 12.0. This will likely shift further as more people are testing.

File: README.md [hunk 1]
```diff
@@ -1,36 +1,45 @@
...
-- The current (11.3-Ux) Freenas Drivers DO NOT support Intel CPU Generation 8 and 9. This means Coffee Lake and later, Intel Xeon E-xxxxG and Intel Core i3/i5/i7 8xxx and later.
-- Support for Generation 8 and 9 can not be added without a kernel upgrade to 12.x. Don't bother trying to compile it from source. TrueNAS 12.x Core will gain driver support for the iGPUs in 8th and 9th gen CPUs.
+- The 11.3-Ux FreeNAS drivers support the iGPU in Intel CPU Generation 2-7, Sandy Bridge through Kaby Lake
+- The 11.3-Ux FreeNAS drivers DO NOT support the iGPU in Intel CPU Generation 8 and 9. This means Coffee Lake and later, Intel Xeon E-xxxxG and Intel Core i3/i5/i7/i9 8xxx and later.
+- Support for the iGPU in generation 8 and 9 CPUs can not be added without a kernel upgrade to 12.x. Preliminary testing shows that TrueNAS Core 12.0 / FreeBSD 12.1 supports the iGPUs in 8th generation CPUs (Coffee Lake i3/5/7/9-8xxx and related Pentium, Xeon E-21xxG), does not support the iGPUs in 9th generation desktop CPUs (Coffee Lake Refresh i3/5/7/9-9xxx and related Pentium), and likely does support the iGPUs in 9th generation entry-level Xeon CPUs (Xeon E-22xxG).
```
Comment (Ornias1993) [inline README.md:18]: Supprt for 9th gen should be in the 12 kernel, 8 and 9th gen drivers support was the same commit...

Comment (Ornias1993) [reply README.md:18]: These are the initial support pushes:
https://github.com/FreeBSDDesktop/kms-drm/commit/1f2bfbcd97c71f919f492dea488b7a4efb8a432e
https://github.com/FreeBSDDesktop/kms-drm/commit/fc623cac8b9bd0e0440fa3a42fe8b0fa16335019

Comment (Ornias1993) [reply README.md:18]: For FreeNAS 12 the drm-kmod is latest: g20200221

**Comment by thorsteneb** [comment]
Take a look at the forum discussion and the release notes for 4.20 and 5.1 kernels. There are two PCI IDs missing in 4.16:  [8086:3e98]  which is the 630 iGPU in i7-9xxx and i9-9xxx, might also be the same for i3 and i5; and [8086:3e9c],  which I am not sure but might be the 9th gen Pentium/Celeron 610 iGPU.

What we have so far is that i915kms.ko does not bring in /dev/dri for 9th gen desktop processors. The kernel release notes for 4.20 jive with that, since that's when that PCI ID came in. FreeBSD 12.x and 13.0 use 4.16; drm-devel for FreeBSD 13.0 uses 5.3 currently. That leads me to believe that once drm-devel comes out of devel and FreeBSD 13.0 is released, we'll have full 9th gen support, and the 10th gen support that's in 5.3. I'm not holding my breath on 11th gen, FreeBSD lags the Linux kernel versions by a bit.

The P630 PCI IDs are in, which is why I am presuming that 9th gen Xeon will work.

"It doesn't work until it's been tested working" - holds true once more, it appears like.

**Pull request merged**